### ISSUE TITLE
Support value template in props definition

### ISSUE THREAD

**Issue by xiajingchun** [vesoft-inc/nebula-importer#287]
**Is your feature request related to a problem? Please describe.**

Right now, only "index" is supported in props definition to map a prop value to a column of csv. Sometimes, this requires a lot of prework to format the csv files. 

**Describe the solution you'd like**

Probably, we can add a "template" to allow more flexible value mapping. 
Say, $i refers to the i th column in the csv.

The simplest template is like below, same as "index";

```
  props:
    - name: "propname"
      type: "xxxxx"
      template: $i
```
But how about below?

`tempalte: left($i, 3)`
`tempalte: $i + $j`
`tempalte: timestamp(datetime($i))`
`tempalte: case when $i==something then $j else $k end`

You can think of more. 

Basically, most of the nGQL functions and also  "case when" can be supported in the templates.
This will use more GraphD CPU, but the usability gain is a lot. 

**Describe alternatives you've considered**

**Additional context**

**Comment by veezhang** [comment]
感觉有些问题需要细化下：
1. 如果有模版，那么其他的字段是否生效？比如： `nullable` 、`nullValue`、`alternativeIndices`、`defaultValue`.
<img width="827" alt="image" src="https://github.com/vesoft-inc/nebula-importer/assets/14001308/1f86846f-08db-441f-ac0e-66822ff17ec3">
2. 模版中，类型怎么处理，比如 $i 是应该是字符串，还是数字？
3. $后面不是数字怎么处理？
4. 如果模版本身包含 $ 字符，怎么处理？ 
5. 启动模板后，还需不需要做类型转换？ 比如 `left($i, 3)` 还要不要加上 "left($i, 3)"，再举一个例子，比如 prefix_$0

**Comment by xiajingchun** [comment]
my 2 cents:
1. template should definitely override index, and other null related settings should be ingored too.
2. no need to deal with the data type, let the user decide, and in this case, it will be `"$i"` when $i is a string, e.g., `left("$i",3)`
